integ-test/src/test/java/org/opensearch/sql/sql/QueryIT.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BEER;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+public class QueryIT extends SQLIntegTestCase {
```

Comment:
Please, add a test which proves that `query` and `query_string` return the same result.
Something like:
```java
var response1 = exec(" ... query('column:text')");
var response2 = exec(" ... query_string(['column'], 'text')");
assertEquals(response1.getJSONArray("datarows"), response2.getJSONArray("datarows"));
```
Pick a query which returns multiple rows.

Reply:
Added in be14171c514ec246ba9d533d989b293595457995
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/143/commits/938440bc1ac8162dbcc703ebec1fc2c07cd871cf